content/kubernetes/active-active/edit-clusters.md

```diff
@@ -75,19 +75,19 @@ To communicate with other clusters, all participating clusters need access to th
 
 1. From one of the existing participating clusters, create a `RedisEnterpriseRemoteCluster` (RERC) custom resource for the new participating cluster.
 
-  This example shows a RERC custom resource for an REC named `rec3` in the namespace `ns3`. 
+  This example shows a RERC custom resource for an REC named `rec-boston` in the namespace `ns-massachusetts`. 
 
   ```yaml
   apiVersion: app.redislabs.com/v1alpha1
   kind: RedisEnterpriseRemoteCluster
   metadata:
-    name: rerc3
+    name: rerc-logan
   spec:
-    recName: rec3
-    recNamespace: ns3
-    apiFqdnUrl: test-example-api-rec3-ns3.redis.com
-    dbFqdnSuffix: -example-cluster-rec3-ns3.redis.com
-    secretName: redis-enterprise-rerc3
+    recName: rec-boston
+    recNamespace: ns-massachusetts
+    apiFqdnUrl: test-example-api-rec-boston-ns-massachusetts.redis.com
+    dbFqdnSuffix: -example-cluster-rec-boston-ns-massachusetts.redis.com
+    secretName: redis-enterprise-rerc-logan
   ```
 
 1. Create the RERC custom resource.
```

Comment:
The numbered steps aren't rendering correctly in the "Create RERC" section. Adding whitespace before any code samples or extra sentences that belong to a step should fix this.